11 - Events/Obsidian October 2021.md

```diff
@@ -46,14 +46,14 @@ We're honored to have invited some veteran developers in the community to be on
 
 ### Plugin judges
 
-- [[Liam Cain]], author of the 3 plugins including **Calendar**
-- [[Matthew Meyers]], author of 8 plugins including **Kanban** and the **California Coast** theme
-- [[Argentina Ortega Sáinz|Argentina Ortega Sáinz (argentum)]], author of **Natural Language Dates** and **Hotkey++**
+- [[Liam Cain]], author of the 3 plugins including the popular [[calendar|Calendar]] plugin
+- [[Matthew Meyers]], author of 8 plugins including the [[obsidian-kanban|Kanban]] plugin and the [[California Coast]] theme
+- [[Argentina Ortega Sáinz|Argentina Ortega Sáinz (argentum)]], author of the [[nldates-obsidian|Natural Language Dates]] and [[hotkeysplus-obsidian|Hotkeys++]]
 
 ### Theme judges
 
-- [[Stephan Ango|Stephan Ango (kepano)]], author of **Minimal** and 4 plugins
-- [[Chetachi Ezikeuzor|Chetachi Ezikeuzor (Chetachi)]], author of **Yin and Yang** and **cMenu**
-- [[SlRvb]], author of **ITS**
+- [[Stephan Ango|Stephan Ango (kepano)]], author of [[Minimal|Minimal]] theme and 4 plugins
+- [[Chetachi Ezikeuzor|Chetachi Ezikeuzor (Chetachi)]], author of [[Yin and Yang|Yin and Yang]] and [[cmenu-plugin|cMenu]]
```

Comment:
I think we don't need the alias for `Yin and Yang` and after it `theme` should be added, after cMenu `plugin`.

And should we add author of `the` Minimal theme and author of `the` Yin and ... and `the` cMenu plugin?